state-sync/diem-data-client/src/diemnet/tests.rs

```diff
@@ -220,46 +220,46 @@ async fn bad_peer_is_eventually_banned_internal() {
     let good_peer = mock_network.add_connected_peer();
     let bad_peer = mock_network.add_connected_peer();
 
-    // bypass poller and just add the storage summaries directly
+    // Bypass poller and just add the storage summaries directly.
```

Comment:
There's a small part of me that gets very excited by nicely formatted comments, lol 😄